templates/web/bristol/report/new/roads_message.html

```diff
@@ -0,0 +1,8 @@
+<div id="bristol-not-owned" class="hidden js-responsibility-message js-roads-bristol">
```

Comment:
Could this (and other template change) be a `no_asset_message` in the configuration, or is there a reason that didn't work?